templates/new_service/terraform/application/config/review.tfvars.json

```diff
@@ -0,0 +1,7 @@
+{
+    "cluster": "test",
+    "namespace": "#NAMESPACE_PREFIX#-development",
+    "environment": "development",
```

Comment:
environment is dynamic for review apps so it should not be here

Reply:
Removed environment